src/routes/write-exercise/components/Actions.tsx

```diff
@@ -1,42 +1,43 @@
 import React, { ReactElement } from 'react'
-import { Text, StyleSheet } from 'react-native'
 import { WhiteNextArrow, NextArrow } from '../../../../assets/images'
 import Button from '../../../components/Button'
 import { BUTTONS_THEME } from '../../../constants/data'
 import { COLORS } from '../../../constants/colors'
 import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
 import labels from '../../../constants/labels.json'
+import styled from 'styled-components/native'
 
-export const styles = StyleSheet.create({
-  lightLabel: {
-    textAlign: 'center',
-    color: COLORS.lunesWhite,
-    fontFamily: 'SourceSansPro-SemiBold',
-    fontSize: wp('4%'),
-    letterSpacing: 0.4,
-    textTransform: 'uppercase',
-    fontWeight: '600'
-  },
-  darkLabel: {
-    textAlign: 'center',
-    color: COLORS.lunesBlack,
-    fontFamily: 'SourceSansPro-SemiBold',
-    fontSize: wp('4%'),
-    letterSpacing: 0.4,
-    textTransform: 'uppercase',
-    fontWeight: '600'
-  },
-  disabledButtonLabel: {
-    color: COLORS.lunesBlackLight
-  },
-  arrowLabel: {
-    marginRight: 8
-  },
-  arrow: {
-    marginLeft: 5
-  }
-})
-
+const LightLabelArrow = styled.Text`
+  text-align: center;
+  color: ${COLORS.lunesWhite};
+  font-family: 'SourceSansPro-SemiBold';
+  font-size: ${wp('4%')};
+  letter-spacing: 0.4;
+  text-transform: uppercase;
+  font-weight: 600;
+  margin-right: 8;
+`
+export const LightLabelInput = styled.Text`
+  text-align: center;
+  font-family: 'SourceSansPro-SemiBold';
+  font-size: ${wp('4%')};
+  letter-spacing: 0.4;
+  text-transform: uppercase;
+  font-weight: 600;
+  color: ${(prop: StyledProps) => (prop.styledInput) ? COLORS.lunesBlackLight : COLORS.lunesWhite};
+`
```

Comment:
I think you can also write:

```suggestion
export const LightLabelInput = styled.Text`
  text-align: center;
  font-family: 'SourceSansPro-SemiBold';
  font-size: ${wp('4%')px};
  letter-spacing: 0.4px;
  text-transform: uppercase;
  font-weight: 600;
  color: ${(prop: StyledProps) => (prop.styledInput) ? COLORS.lunesBlackLight : COLORS.lunesWhite};
`

const LightLabelArrow = styled(LightLabelInput).attrs(props => ({ styledInput: null}))

```